src/Business Foundation/App/AuditCodes/src/ReasonCode.Table.al

```diff
@@ -0,0 +1,55 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Foundation.AuditCodes;
+
+/// <summary>
+/// Reason codes supplement the source codes and are used to indicate why an entry was created.
+/// </summary>
+/// <remarks>
+/// https://learn.microsoft.com/en-us/dynamics365/business-central/finance-setup-trail-codes
+/// </remarks>
+table 231 "Reason Code"
+{
+    Caption = 'Reason Code';
+    LookupPageID = "Reason Codes";
+    DataClassification = CustomerContent;
```

Comment:
@grobyns To be in sync with table 6635 "Return Reason", why not providing DrillDownPageId as well?